docs/SPRIG_APP_REVIEW.md

```diff
@@ -19,6 +19,7 @@
   * Ensure that name of the game file contains only alphanumeric characters (or -, or _), and does not conflict with any other game.
   * Ensure that the game is placed in the /games directory.
   * Ensure that the game screenshot is placed in the /games/img directory, and that the name of the image matches the name of the game file.
+  * Ensure that the author has provided all information requested on header of the code (@title: @author: @tags: [] @img: @addedOn:).
```

Comment:
you might want to remove this step after marios adds the code that will programmatically enforce this (the service won't start up, preview will fail)